### PR TITLE
Update Quick Start section in User Guide

### LINKED ISSUE #28: Update Quick Start section in User Guide

### PR DESCRIPTION
**Pull request by itsme-zeix**
Closes #28

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -5,62 +5,94 @@ title: User Guide
...
+- On Windows, press the **Windows Key** and type **`Command Prompt`** in the search bar. Click on the **Command Prompt** application to open it.
+- On macOS, press **Cmd + Space**, type **`Terminal`** into the search bar, and press **Enter**.
+- On Linux, open your Terminal application.
+
+### 5. Run the applicaton
+
+Navigate your terminal to the folder where you saved the AgentAssist application:
 
-1. Download the latest `.jar` file from [here](https://github.com/se-edu/addressbook-level3/releases).
+- Type **`cd `** (with a space after it), then drag and drop the folder where you placed the `.jar` file into the terminal window. Press Enter.<br>
```
Comment (FionaQY) [inline docs/UserGuide.md:44]: maybe add an example? e.g. `cd samplefolder`

File: docs/UserGuide.md [hunk 2]
```diff
@@ -5,62 +5,94 @@ title: User Guide
...
 
-   * `list` : Lists all contacts.
+In the application window, you will see a **command box**. This is where you type commands to manage your contacts.
 
-   * `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01` : Adds a contact named `John Doe` to the Address Book.
+- Type a command in the command box and press **Enter** to make the app do something.
+- Typing `help` and pressing Enter will open the help window.
 
-   * `delete 3` : Deletes the 3rd contact shown in the current list.
+Here are some commands you can try:
```
Comment (FionaQY) [inline docs/UserGuide.md:58]: Maybe use "example commands" instead of just "commands" to make it clear this list is non-exhaustive

**Review by FionaQY — APPROVED** [review]
LGTM

File: docs/UserGuide.md (hunk 1 above)
Comment (itsme-zeix) [reply docs/UserGuide.md]: Done! Give it a check.

File: docs/UserGuide.md (hunk 2 above)
Comment (itsme-zeix) [reply docs/UserGuide.md]: Agreed, this does clarify the meaning too. I have included the change.

**Pull request merged**